docs/specs/markdown.yml

```diff
@@ -17,6 +17,24 @@ outputs:
     { "content": "<p><em>this</em> <em>is</em> an <em><strong>inline</strong></em> <code>markdown</code> <strong>test</strong></p>" }
   build.manifest:
 ---
+# Report warning when Yaml header has null value
+inputs:
+  docfx.yml:
+  docs/a.md: |
+    ---
+    title:
+    ---
+    hello
+outputs:
+  docs/a.json: |
```

Comment:
how about leave `docs/a.json: ` as empty, to just check build.log

Reply:
sounds good